schemas/data/metrics.schema.json

```diff
@@ -75,6 +75,15 @@
         },
         "https://ns.adobe.com/xdm/data/metrics/direct-marketing/user-complaints": {
           "$ref": "https://ns.adobe.com/xdm/data/measure"
+        },
+        "https://ns.adobe.com/xdm/data/metrics/direct-marketing/opens": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure"
+        },
+        "https://ns.adobe.com/xdm/data/metrics/direct-marketing/unsubscriptions": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure"
+        },
+        "https://ns.adobe.com/xdm/data/metrics/direct-marketing/not-sent": {
+          "$ref": "https://ns.adobe.com/xdm/data/measure"
```

Comment:
Please provide descriptions for all of these simular to https://github.com/adobe/xdm/pull/223. It would actually be good if the metrics were in their own PR to make this more focussed.

Reply:
Adding descriptions. Too much to split the PR now. Next time maybe?